Update Readme (#38)
Worker20/CreateRobotics: README.md

<a href="https://discord.gg/PbBt4PQhpe"><img src="https://discordapp.com/api/guilds/865321790347018241/widget.png?style=banner3" alt="Discord Banner" align="right"/></a>

> **Create Robotics** is a create addon that adds robotics and their automation to Minecraft; 

Reviewer: Simple change, replace the `;` with a `.`
```suggestion
**Create Robotics** is a create addon that adds robotics and their automation to Minecraft. 
```

> players can program the robots with {Placeholder name}, designed to be user-friendly, Simple, But still powerful. 

Reviewer: ```suggestion
Players can program the robots with our own scripting-language called RoboScript, designed to be user-friendly, simple, but still powerful.
```

This allows the player to do many things with the robots, whether automating moving items from A to B, building a quarry, or simply using them to automate a mob grinder. 

> The mod adds almost infinite ways to automate with the robots." 

Reviewer: Why a `"` at the end?
```suggestion
The mod adds almost infinite ways to automate with the robots!
```

> The syntax for {Placeholder name} can be found <a href="https://github.com/Worker20/CreateRobotics/wiki/Programming-Mechanic">here.</a>

Reviewer: ```suggestion
For help with RoboScript you can look <a href="https://github.com/Worker20/CreateRobotics/wiki/Programming-Mechanic">here</a>.
```
